### DISCUSSION TITLE
posts/2022/09/keep-it-canonical-seo/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#46] [Announcements]
# posts/2022/09/keep-it-canonical-seo/

Use canonical URLs to keep search engines straight on which content they should index.

https://www.brycewray.com/posts/2022/09/keep-it-canonical-seo/

**Comment by avionics18** [comment]
Thanks for explaining it in so simple terms. There is so much trash on youtube, just got a headache.

**Reply by brycewray** [reply]
@avionics18 Glad it was helpful! And your statement about YouTube is true about **many** aspects of it, but especially the SEO-related content.
